flatdata-cpp/include/flatdata/MultiVector.h

```diff
@@ -113,14 +139,15 @@ template < typename IndexType, typename... Args >
 void
 MultiVector< IndexType, Args... >::add_to_index( )
 {
-    m_index.grow( ).value = m_size;
+    m_index.grow( ).value = m_size_serialized + m_data.size( ) - PADDING_SIZE;
```

Comment:
In future might it be useful to encapsulate `PADDING_SIZE` somewhere like "padded byte array" as this `-
 PADDING_SIZE` is always there and is quite easy to miss....